tests/unit/babylon/src/Loading/babylon.sceneLoader.tests.ts

```diff
@@ -731,4 +731,94 @@ describe('Babylon Scene Loader', function() {
             return BABYLON.SceneLoader.LoadAsync("file:///", "Box.gltf").then(resetPreprocessUrl, resetPreprocessUrl);
         });
     });
-});
+
+    describe('#DirectLoad', () => {
+        it('should load a base64 encoded obj with no mime type', () => {
+            const scene = new BABYLON.Scene(subject);
+            return BABYLON.SceneLoader.ImportMeshAsync('', 'data:;base64,' + objBase64, undefined, scene, undefined, ".obj").then((result) => {

```

Comment:
very nit-picky, only if you are making changes to the PR:

```suggestion
            return BABYLON.SceneLoader.ImportMeshAsync('', `data:;base64,${objBase64}`, undefined, scene, undefined, ".obj").then((result) => {

```

This is the same for the entire file (9 more instances)